pkgs/build-support/trivial-builders.nix

```
@@ -250,6 +250,26 @@ rec {
       '';
     };
 
+  /*
+   * Writes a beautified and validated, syntax checked sql file /nix/store/<store path>.sql
+   * via checkPhase
+   *
+   * Example:
+   * writePSQL "foo" "CREATE DATABASE my_api;"
+   * }
+  */
+  writePSQL = name: text:
+    writeTextFile {
+      name = "${name}.sql";
+      inherit text;
+      checkPhase =
+        ''
+          runHook preCheck
+          ${pgformatter}/bin/pg_format "$target"
+          runHook postCheck
+        '';
```

```suggestion
      inherit text;
      checkPhase = ''
        runHook preCheck
        ${buildPackages.pgformatter}/bin/pg_format "$target"
        runHook postCheck
      '';
```
Don't we require nativeBuildInputs (which would require runCommand) to support cross?